build/images/tf_operator/Dockerfile

```diff
@@ -1,21 +1,13 @@
-FROM golang:1.10 AS build-image
+FROM golang:1.13.5 AS build-image
 
 ADD . /go/src/github.com/kubeflow/tf-operator
 
 WORKDIR /go/src/github.com/kubeflow/tf-operator
 
 RUN go build -o tf-operator.v1 ./cmd/tf-operator.v1
 
-FROM registry.access.redhat.com/ubi8/ubi:latest
+FROM gcr.io/distroless/base-debian10
 
-# TODO(jlewi): We should probably change the directory to /opt/kubeflow.
-RUN mkdir -p /opt/kubeflow/samples
+COPY --from=build-image /go/src/github.com/kubeflow/tf-operator/tf-operator.v1 /opt/
 
-COPY tf_smoke.py /opt/kubeflow/samples/
-RUN chmod a+x /opt/kubeflow/samples/*
-
-COPY --from=build-image /go/src/github.com/kubeflow/tf-operator/tf-operator.v1 /opt/kubeflow
-
-RUN chmod a+x /opt/kubeflow/tf-operator.v1
-
-ENTRYPOINT ["/opt/kubeflow/tf-operator.v1"]
+ENTRYPOINT ["/opt/tf-operator.v1"]
```

Comment:
Probable permission issue.

Reply:
Good question. Here's the pattern for moving binaries to a distroless image given by the example:

https://github.com/GoogleContainerTools/distroless/blob/5a460b2310a3e2e3c0590835614860747d6e8769/examples/go/Dockerfile#L10-L12

The idea here is to keep the distroless image with just the essentials for golang and its binaries. That leaves this system with just the user who's running this binary. So I guess setting permissions isn't necessary as advocated by the example.